docs/automation.rst

```diff
@@ -0,0 +1,148 @@
+Rollup Automation
+=================
+
+Once you get comfortable with manual rollups, you might want to set up
+regularly executed automated rollups.
+
+At this moment scaraplate doesn't provide a CLI for that, but there's
+a quite extensible Python code which simplifies implementation of custom
+scenarios.
+
+Supported automation scenarios
+------------------------------
+
+GitLab Merge Request
+++++++++++++++++++++
+
+GitLab integration requires `python-gitlab <https://python-gitlab.readthedocs.io>`_
+package, which can be installed with:
+
+::
+
+    pip install 'scaraplate[gitlab]'
+
+
+Sample ``rollup.py`` script:
```

Comment:
Generally, it feels like that you don't need any code to run autorollups, but some ini / yaml file with configuration. Have you thought about something like this?

Reply:
Yep, and a CLI around that. That would be preferable, but I suppose it is a subject for future work. This is already too bloated as it is.

Reply:
Cool then. Let's see how it goes. Code is more flexible than ini file.